sklearn/grid_search.py

```diff
@@ -652,6 +652,9 @@ class GridSearchCV(BaseSearchCV):
     n_jobs : int, default=1
         Number of jobs to run in parallel.
 
+        .. versionchanged:: 0.17
```

Comment:
I think you should not add this here. My comment was that if you added it here, you would need to add it to all n_job calls. I don't think that makes sense.